aws/hll/aws-s3-transfer-manager/src/download.rs

```diff
@@ -34,17 +49,183 @@ impl From<GetObjectInputBuilder> for DownloadRequest {
     }
 }
 
-/// Response type for a single download object request.
-#[derive(Debug)]
-#[non_exhaustive]
-pub struct DownloadResponse {
-    /// Object metadata
-    pub object_meta: ObjectMetadata,
+/// Fluent style builder for [Downloader]
+#[derive(Debug, Clone)]
+pub struct Builder {
+    target_part_size_bytes: u64,
+    checksum_validation_enabled: bool,
```

Comment:
aws-c-s3 also started with options like this on the "Client", but when integrating with Java and Python SDKs we ended up needing to make them configurable per "meta-request" as well.

I know this PR is already merged. I just remembered...